### PR TITLE
Tweak networking thread names

### PR DESCRIPTION
**Pull request by jukkar**
Makes the "kernel stacks" command look nicer.

### CODE REVIEW

**Review by tbursztyka — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

File: subsys/net/ip/net_tc.c [hunk 1]
```diff
@@ -18,6 +18,8 @@ LOG_MODULE_REGISTER(net_tc, CONFIG_NET_TC_LOG_LEVEL);
 #include "net_stats.h"
 #include "net_tc_mapping.h"
 
+#define MAX_NAME_LEN sizeof("xx_q[y]")
```
Comment (galak) [inline subsys/net/ip/net_tc.c:21]: Feels like a comment here about what 'xx' means would be good.

**Review by galak — DISMISSED** [review]

File: subsys/net/ip/net_tc.c (hunk 1 above)
Comment (jukkar) [reply subsys/net/ip/net_tc.c:21]: Just a normal design pattern to calculate the output string length. The "xx" in this context means either "rx" or "tx" in the output string.

**Comment by jukkar** [comment]
Updated according to comments.

**Comment by jukkar** [comment]
ping @galak, can you recheck

**Comment by jukkar** [comment]
ping @galak

**Comment by jukkar** [comment]
ping @galak, could you re-check so we get this merged

**Comment by jukkar** [comment]
ping @galak

**Pull request merged**